source/mbed-util_mbed.c

```diff
@@ -21,12 +21,12 @@
 #include <stdarg.h>
 #include "device.h"
 #include "mbed/mbed_interface.h"
-#include "mbed-util/mbed-util.h"
+#include "core-util/core-util.h"
```

Comment:
I think the last thing left to do here is to rename `mbed-util_mbed.c` to `core-util_mbed.c` (or even `core-util_mbed_os.c`)